libraries/shared/executor/executor_test.go

```diff
@@ -0,0 +1,72 @@
+package executor_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/executor"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/mocks"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/watcher"
+	"github.com/vulcanize/vulcanizedb/pkg/core"
+	"github.com/vulcanize/vulcanizedb/pkg/datastore/postgres"
+	"github.com/vulcanize/vulcanizedb/pkg/fakes"
+	"github.com/vulcanize/vulcanizedb/test_config"
+	"time"
+)
+
+var _ = Describe("Executor", func() {
+	var (
+		db         *postgres.DB
+		blockChain *fakes.MockBlockChain
+		mockPlugin fakes.MockPlugin
+	)
+	BeforeEach(func() {
+		db = test_config.NewTestDB(core.Node{ID: "testNode"})
+		blockChain = fakes.NewMockBlockChain()
+		mockPlugin = fakes.NewMockPlugin()
+	})
+
+	It("adds transformer sets to the executor", func() {
+		ew := watcher.NewEventWatcher(db, blockChain, false, time.Second)
```

Comment:
There's no reason to use the real watcher structs here as opposed to the mocks, so let me know if you think I should use the mocks instead